### ISSUE TITLE
Unexpected Error

### ISSUE THREAD

**Issue by tempus2016** [Andre0512/hon#17]
Spoke too soon 😂

```
This error originated from a custom integration.

Logger: custom_components.hon.hon
Source: custom_components/hon/hon.py:52 
Integration: Haier hOn (documentation, issues) 
First occurred: 03:50:02 (1354 occurrences) 
Last logged: 09:28:15

Unexpected error fetching a8-48-fa-18-ef-e4 data: object _AsyncGeneratorContextManager can't be used in 'await' expression
Unexpected error fetching 78-e3-6d-8d-cc-68 data: object _AsyncGeneratorContextManager can't be used in 'await' expression
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 250, in _async_refresh
    self.data = await self._async_update_data()
  File "/config/custom_components/hon/hon.py", line 52, in _async_update_data
    await self._device.update()
  File "/usr/local/lib/python3.10/site-packages/pyhon/appliance.py", line 162, in update
    await self.load_attributes()
  File "/usr/local/lib/python3.10/site-packages/pyhon/appliance.py", line 154, in load_attributes
    self._attributes = await self._api.load_attributes(self)
  File "/usr/local/lib/python3.10/site-packages/pyhon/connection/api.py", line 85, in load_attributes
    async with self._hon.get(url, params=params) as response:
  File "/usr/local/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/usr/local/lib/python3.10/site-packages/pyhon/connection/handler.py", line 34, in get
    async with self._intercept(self._session.get, *args, **kwargs) as response:
  File "/usr/local/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/usr/local/lib/python3.10/site-packages/pyhon/connection/handler.py", line 86, in _intercept
    yield await self._intercept(method, *args, loop=loop + 1, **kwargs)
TypeError: object _AsyncGeneratorContextManager can't be used in 'await' expression

```

**Comment by Andre0512** [comment]
sh*t, this error comes from the new session refresh method for Andre0512/pyhon#10. 
All `v0.5.0-beta.x` versions should be affected every 8 hours, I will fix this soon.
As workaround you can reload the integration under "devices & services".

**Comment by Andre0512** [comment]
Hey, should be fixed in new release `v0.5.0` :slightly_smiling_face:
